Data management - additional statements (#253)
sa-tre/satre-specification: docs/source/pillars/data_management.md

    For example, a user may take pictures of their computer screen to remove data, or use a device presenting as a USB HID keyboard to input large amounts of text.
    An example of a reasonable measure would be for a remote desktop based TRE to prevent data being copied from a local machine's clipboard to a workspace.
  - Mandatory
* - 3.1.X.
  - Data held within the TRE should be the minimum required for analysis or research.
  - Data stored and processed within the TRE should be limited to the amount required for that purpose.

>     This increases the level of protection for {ref}`data subjects <public_roles>`, makes it easier to comply with data protection legislation (e.g. GDPR) and could reduce the overhead of storage and processing.

Reviewer: ```suggestion
    This increases the level of protection for {ref}`data subjects <public_roles>`, makes it easier to comply with data protection legislation and could reduce the overhead of storage and processing.
```
I don't think we need to particularly call out GDPR.

  - Recommended
```

## Data Archiving

The practice of storing data that is no longer actively used but needs to be retained for historical or compliance reasons.

```{list-table}
:header-rows: 1
:name: tab-data-archiving

* -
  - Statement
  - Guidance
  - Importance
* - 3.X.X.

>   - Archive data within the TRE should be read only.
>   - Archive data by its very nature should not change and therefore be maintained as a read only store.

Reviewer: ```suggestion
  - Archived data within the TRE should be read only.
  - Archived data by its very nature should not change and therefore be maintained as a read only store.
```

    If an update is required, it may be pulled from archive into a separate operational store.
  - Recommended
* - 3.X.X.
  - Long-term archives must be held in simple, standard formats to ensure accessibility.
  - Some data archives may be required by policy or legislation to be kept for very long periods within the scope of the TRE.

>     Such data should be held in the simplest possible file format conforming to international standards to ensure they are platform and application agnostic.

Reviewer: ```suggestion
    Such data should be held in the simplest possible file format, conforming to international standards if available, to ensure they are platform and application agnostic.
```

    If egress of such files is permitted then the risks of inadvertent disclosure must be mitigated and documented.
    Refusing to allow egress of such files is also a valid policy decision.
  - Mandatory
* - 3.3.6
  - You should have a statistical basis to guide the decisions of an output checker on the safety of outputs.

>   - There should be a solid basis to allow decision to be made about data based on risk factors such as re-identification of an individual or risk to commercial operations posed by outputs from the TRE.

Reviewer: ```suggestion
  - There should be a solid basis to allow decisions to be made about data based on risk factors such as re-identification of an individual or risk to commercial operations posed by outputs from the TRE.
```

    This reduces the number of unique configurations that need to be supported.
  - Optional
```

## Research Meta-Data

Descriptive information about research data, helping researchers understand and manage the data effectively.

```{list-table}
:header-rows: 1
:name: tab-meta-data

* -
  - Statement
  - Guidance
  - Importance
* - 3.7.1.
  - You should have a consistent and easily accessible meta-data data model or similar to describe what a data asset contains.

>   - Where possible existing data models should be employed potentially extended.

Reviewer: ```suggestion
  - Where possible existing data models should be employed, and extended if necessary.
```

    More detailed information on the data schema for data assets should also be provided to assist researchers in understanding what data may be available without the need to see the underlying data.
  - Recommended
* - 3.7.2.
  - You could provide summary, abstracted or synthetic data to researchers without exposing the underlying data set.

>   - To reduce the need for access to row level data researchers could be provided with non-sensitive versions of the data either as summary data or using synthetic version of the data for activities such as code development and cohort planning.

Reviewer: ```suggestion
  - To reduce the need for access to row level data researchers could be provided with non-sensitive versions of the data either as summary data or using synthetic versions of the data for activities such as code development and cohort planning.
```
